app/assets/javascripts/MeasurePackage.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const [MongoBuffer, ObjectId] = [
+  mongoose.Schema.Types.Buffer,
+  mongoose.Schema.Types.ObjectId,
+];
+
+const MeasurePackageSchema = mongoose.Schema(
+  {
+    file: MongoBuffer,
```

Comment:
I'm not 100% sure what this is for, are we saving the actual measure `zip` to our database?

Reply:
This was taken straight out of Bonnie's Measure code, so it looks like it.

Reply:
https://github.com/projecttacoma/bonnie_bundler/blob/master/lib/models/cql_measure.rb#L154

Reply:
Hmm, okay. I'm not huge on saving measures right into our mongo database, rather than using something like GridFS, or even just using the filesystem. I'd like to better understand why this was done in the first place. I'll look into this myself and get back to you.

Reply:
Yes, this was implemented so we could use the actual measure package for rebuilding the elm and for bug investigation purposes.